PocketKit/Sources/PocketKit/Home/SlateDetailViewController.swift

```diff
@@ -51,6 +51,7 @@ class SlateDetailViewController: UIViewController {
 
         view.accessibilityIdentifier = "slate-detail"
         navigationItem.title = model.slateName
+        hidesBottomBarWhenPushed = true
```

Comment:
what's this change for?

Reply:
When we push the slate detail, we want to hide the bottom tab bar. I'm not sure when that stopped happening, but I noticed it while I was testing things out so I added this line back.